build/args/all.gn

```diff
@@ -13,3 +13,7 @@ ffmpeg_branding = "Chrome"
 enable_basic_printing = true
 
 is_cfi = false
+
+# FIXME(deepak1556): workaround for https://crbug.com/924225
+# remove this when clang roll 352138 lands.
+enable_precompiled_headers = false
```

Comment:
can we remove this now?

Reply:
The change got landed in 74 https://chromium-review.googlesource.com/c/chromium/src/+/1436036